internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -97,12 +99,109 @@ type SingleUseConnector struct {
 	backendConn net.Conn
 	clientConn  net.Conn
 	logger      log.Logger
+	// Note: We're following standard ctor naming practices with this field.
+	newMSSQLConnector types.MSSQLConnectorCtor
+	readPrelogin      types.ReadPreloginFunc
+	writePrelogin     types.WritePreloginFunc
+	readLogin         types.ReadLoginFunc
+	newTdsBuffer      types.TdsBufferCtor
+}
+
+// NewSingleUseConnector creates a new production SingleUseConnector
+func NewSingleUseConnector(logger log.Logger) *SingleUseConnector {
+	return NewSingleUseConnectorWithOptions(
+		logger,
+		NewMSSQLConnector,
+		ReadPreloginWithPacketType,
+		WritePreloginWithPacketType,
+		ReadLogin,
+		NewTdsBuffer,
+	)
+}
+
+// NewMSSQLConnector is the production implementation of MSSQLConnectorCtor,
+// used for creating mssql.Connector instances.  We need to wrap the raw
+// constructor provided by mssql (ie, mssql.NewConnector) in this function so
```

Comment:
s/ie,/i.e./g (there's a few other occurrences)

not sure if i'm being pedantic and what you have here is a legitimate other way of representing "id est".